lib/content_format_inspector.rb

```diff
@@ -3,7 +3,7 @@ class ContentFormatInspector
 
   attr_reader :error
 
-  MIGRATED_SCHEMAS = %w(answer help_page local_transaction).freeze
+  MIGRATED_SCHEMAS = %w(answer help_page guide local_transaction).freeze
```

Comment:
🔺 pedantry alert 🔺  

'guide' should be before 'help_page' if we're sorting alphabetically 😁